src/protocol.h

```diff
@@ -78,6 +78,7 @@
 #define PROTMESSID_CLM_REQ_CONN_CLIENTS_LIST  1014 // request the connected clients list
 #define PROTMESSID_CLM_CHANNEL_LEVEL_LIST     1015 // channel level list
 #define PROTMESSID_CLM_REGISTER_SERVER_RESP   1016 // status of server registration request
+#define PROTMESSID_CLM_RECORDER_STATE_CHANGED 1017 // new recording state of server
```

Comment:
This is not good. Why is this a connection less message? That must certainly a normal message instead.

Reply:
It happens at any time under the person sitting at the server window's control.  It doesn't relate to anything any client did.

Reply:
But it only only applies to all connected clients. If they are connected, they can receive normal messages.

Reply:
If you use connection less messages, the message can get lost and the indicator is incorrect. The normal message makes sure the indicator is always correct.

Reply:
See question on other comment.